scripts/bump-release

```diff
@@ -8,12 +8,18 @@
 #
 # YMMV, no disclaimer or warranty, etc.
 
-if ! [[ "$1" =~ ^v[[:digit:]]+.[[:digit:]]+.[[:digit:]]$ ]]; then 
+if ! [[ "$1" =~ ^v[[:digit:]]+.[[:digit:]]+.[[:digit:]]$ ]]; then
 	echo "Usage: scripts/bump-release <VERSION>"
 	echo "   where VERSION must be vX.Y.Z"
 	exit 255
 fi
 
+# make sure we are running in a toplevel directory
+if ! [[ -d '.git' ]]; then
```

Comment:
Not sure if I understand. What would you do with `$0`? Expect it to be `./scripts/bump-release` or `scripts/bump-release`?

I can check for `configure.ac` instead of `.git`.